### PR TITLE
rename scenario_test_runner_launch to scenario_simulator_launch

### PR DESCRIPTION
**Pull request by danielm1405**
## Types of PR

- [ ] New Features
- [X] Upgrade of existing features
- [ ] Bugfix

## Description

Due to change of package name (proposed [here](https://gitlab.com/robotecai-public/AutowareAuto/-/merge_requests/9)) `awf/auto` launch package needs to be changed as well.

### CODE REVIEW

File: test_runner/scenario_test_runner/launch/scenario_test_runner.launch.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def default_autoware_launch_package_of(architecture_type):
             f"architecture_type := {architecture_type.perform(context)} is not supported. Choose one of {architecture_types()}."
         )
     return {
-        "awf/auto": "scenario_test_runner_launch",
```
Comment (hakuturu583) [inline test_runner/scenario_test_runner/launch/scenario_test_runner.launch.py:43]: Please let me know where is the scenario_test_runner package is?

File: test_runner/scenario_test_runner/package.xml [hunk 2]
```diff
@@ -18,7 +18,7 @@
   <exec_depend>openscenario_utility</exec_depend>
 
   <!-- AutowareAuto launch -->
-  <!-- <exec_depend>scenario_test_runner_launch</exec_depend> -->
```
Comment (hakuturu583) [inline test_runner/scenario_test_runner/package.xml:21]: Please let me know where is the scenario_test_runner package is?

**Review by hakuturu583 — COMMENTED** [review]
Can you tell me where is

File: test_runner/scenario_test_runner/launch/scenario_test_runner.launch.py (hunk 1 above)
Comment (danielm1405) [reply test_runner/scenario_test_runner/launch/scenario_test_runner.launch.py:43]: It is [here](https://gitlab.com/robotecai-public/AutowareAuto/-/merge_requests/9), to be merged with `scenario_simulator_v2_changes` branch

File: test_runner/scenario_test_runner/package.xml (hunk 2 above)
Comment (danielm1405) [reply test_runner/scenario_test_runner/package.xml:21]: It is [here](https://gitlab.com/robotecai-public/AutowareAuto/-/merge_requests/9), to be merged with `scenario_simulator_v2_changes` branch

**Comment by yamacir-kit** [comment]
@danielm1405 
I have heard about the situation from Tsutake -san.
I approve this PR change.

**Review by yamacir-kit — APPROVED** [review]

**Pull request merged**